.tool-versions

```diff
@@ -3,7 +3,7 @@
 nodejs 18.19.0
 elixir 1.15.7-otp-26
 erlang 26.1.2
-terraform 1.6.6
+#terraform 1.6.6
```

Comment:
Is this change intentional?

Reply:
Ah, good catch.  Nope, was having some issues with `asdf` and was just removing that from the equation.  Will get this put back to what it was.

Reply:
Got this changed back to what's in main.